app/main.py

```diff
@@ -1,2 +1,75 @@
+from dataclasses import dataclass
+from typing import Hashable, Any
+
+
+@dataclass
+class Node:
+    key: Hashable
+    value: Any
+    key_hash: int
+
+
 class Dictionary:
     pass
+    INITIAL_CAPACITY = 8
+    THRESHOLD = 2 / 3
```

Comment:
The `THRESHOLD` is currently set to `2 / 3`, which is an integer division in Python 2. In Python 3, this will result in a float, which is correct. Ensure that this is intended to be a float representing the load factor threshold.